website/README.md

```diff
@@ -0,0 +1,17 @@
+# MetalLB Documentation Website
+
+## Setup
+
+Please follow Hugo's official website documentation to install.
+
+The currently used version of Hugo is defined in the [`netlify.toml`](../netlify.toml) configuration file.
+
+## Running the site locally
+
+If you want to develop the site locally, you can run a single command (assuming that you've run the [setup](#setup)):
+
+```bash
+$ make develop
```

Comment:
we don't use make here (unless I am missing something). As I wrote above, I'd remove this README and extend the website contributing guidelines when we implement https://github.com/metallb/metallb/issues/2382

Reply:
Removed this README.md.